types/material-ui-phone-number/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for material-ui-phone-number 2.2
+// Project: https://github.com/alexplumb/material-ui-phone-number
+// Definitions by: Bassem (NextGLabs) <https://github.com/nextglabs>
+//                 Gelsoh <https://github.com/gelzoh>
+//                 Láďa Durchánek <https://github.com/durchanek>
+//                 Nifanic <https://github.com/nifanic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
```

Comment:
```suggestion
// Minimum TypeScript Version: 4.1
```
Why do you need this version? This limits possible interop.
The material-ui itself is at 3.8:
https://github.com/mui-org/material-ui/blob/master/package.json#L146

Reply:
Right, I changed it to 3.8 👍🏻